polyply/src/gen_coords.py

```diff
@@ -37,21 +49,81 @@ def gen_coords(args):
                    'connected by bonds, constraints or virual-sites')
             raise IOError(msg.format(molecule.name))
 
+    if args.split:
+       wrapper = partial(split_residues, split=args.split)
+       pool = multiprocessing.Pool(args.nproc)
+       pool.map(wrapper, tqdm(topology.molecules))
```

Comment:
Somehow this parallelizes the statement but molecules are not updated in place as used to. I suppose this has something todo with python pickling the function. @pckroon any idea?

Reply:
removed this for now. Splitting is still super slow because it involves making the residue graph multiple times. But mehh issue for another time

Reply:
for now I removed this because it cannot be parallelized easily